README.md

```
@@ -3,13 +3,41 @@ Firmware for Brain-BioAmp hardware from Upside Down Labs
+|5 | [R4BCIToggle](5_BCI_Toggle)| Toggles the LED on/off using sustained focus for 4 seconds. (both R4 Minima and R4 WiFi can be used)|
+|6 | [R4BCISpiral](6_BCI_Spiral)| Runs the spiral LED game based on focus. (Only R4 WiFi can be used)
 
 ## Examples
 
 1. **EEG Filter**
 
     A band-pass filter for EEG signals between 0,5 Hz and 44.5 Hz 
 
-    <img src="2_EEGFilter/EEGFilter.png" height="300" width="400">
\ No newline at end of file
+    <img src="2_EEGFilter/EEG_Filter.png" height="300" width="400">
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)